frontend/src/metabase/core/components/Link/Link.tsx

```diff
@@ -19,14 +24,23 @@ const Link = ({
   children,
   disabled,
   tooltip,
+  tabIndex,
   ...props
 }: LinkProps): JSX.Element => {
+  const computedTabIndex = useMemo(() => {
```

Comment:
`useMemo` won't give any performance gains here, and actually it would decrease performance. Could it be removed?